.travis.yml

```diff
@@ -11,11 +15,10 @@ stages:
 
 jobs:
   include:
-    - stage: validate
-      name: "TypeScript Compile"
-      script: npm run clean && npm run build
```

Comment:
so when would TS compile happen now?

Reply:
Currently already happens post-install on NPM from the root - so I removed this as it'd be doing it twice otherwise.

https://github.com/firebase/extensions/pull/86 would additionally add TS compile to pre-commit also to ensure compiled code is up to date before committing.

Reply:
Got it.